src/itest/java/uk/gov/companieshouse/company/links/steps/StreamConsumerSteps.java

```diff
@@ -223,24 +229,40 @@ private ResourceChangedData createMessageWithInvalidEventType() {
     }
 
     private void initialiseVariablesToDeltaType() {
-        mainTopic = String.format("stream-company-%s", deltaType);
+        if (deltaType.equals("statements")) {
+            mainTopic = "stream-psc-statements";
+            resourceUri = String.format("company/%s/persons-with-significant-control-statements", COMPANY_NUMBER);
+        } else {
+            mainTopic = String.format("stream-company-%s", deltaType);
+            resourceUri = String.format("company/%s/%s", COMPANY_NUMBER, deltaType);
+        }
         topicPrefix = String.format("%s-company-links-consumer", mainTopic);
-        resourceUri = String.format("company/%s/%s", COMPANY_NUMBER, deltaType);
     }
 
```

Comment:
Small point but the function name doesn't make sense any more

Reply:
Fair, I guess initialiseVariablesUsingDeltaType?

Reply:
Yep that works